doc/api/deprecations.md

```diff
@@ -660,6 +660,14 @@ Type: Runtime
 
 `REPLServer.parseREPLKeyword()` was removed from userland visibility.
 
+<a id="DEP0076"></a>
+### DEP0076: Module.\_debug()
```

Comment:
This should actually be `DEP00XX` until it lands. The person landing needs to assign the specific code.

Reply:
And the 

```js
Module._debug = util.deprecate(debug, 'Module._debug is deprecated.',
                               'DEP0076');
```
should be DEP00XX also?

Reply:
Yeah, all of the `DEP00..` identifiers.